### PR TITLE
Add benchmark CI job for fast checks

### LINKED ISSUE #1268: Add CI job for benchmarks
### Which part of the code is the issue addressing?

* CI Feature

### Description

Run tests benchmarks and run runtimes benchmarks are very different. We need to be sure the runtime benchmarks are not broken when we make a change.

Add a CI job that runs all runtime benchmarks with one iteration.

### How will this affect the code base

- Reliability of `main` code

### What are forseen obstacles or hurdles to overcome?

- This CI job can be very long. We need to keep as simple as possible.

### PR DESCRIPTION
**Pull request by lemunozm**
# Description

This job does not generate weights, it only checks fastly for all runtimes if benchmarks work/compile.

You can test the script locally as follows for fast benchmark checks (it takes around 10-15s to check all runtime benchmarks once it's compiled):

```sh
./scripts/check_benchmarks.sh <runtime>
```

Being `<runtime>` the following possible values: `development`, `altair`, `centrifuge`

Fixes #1268

## Changes and Descriptions

- New script
- New CI job

### CODE REVIEW

**Review by mustermeiszer — APPROVED** [review]
Looks good to me! But given we change CI I would also wait for @gpmayorga approval

**Review by NunoAlexandre — APPROVED** [review]
LGTM 🌞

**Comment by lemunozm** [comment]
This PR fails as expected due to the `pool-registry` benchmark issue solved in #1233

Once that PR is merged, this can be safety merged.

**Comment by lemunozm** [comment]
I think we can run better the three runtimes in the same job to save 1 hour per CI job

**Review by mustermeiszer — APPROVED** [review]
LGTM!

**Pull request merged**
